### PR TITLE
Add guidance on freeing up disk space more quickly

### PR DESCRIPTION
**Pull request by rmloveland**
Fixes DOC-8242, DOC-9641

Actual new content is here:

[operational FAQs](https://deploy-preview-18235--cockroachdb-docs.netlify.app/docs/v23.2/operational-faqs.html#why-is-my-disk-usage-not-decreasing-after-deleting-data)

the other changes are mainly links from other places to the above

### CODE REVIEW

**Review by jbowens — APPROVED** [review]
LGTM!

File: src/current/v23.2/operational-faqs.md [hunk 1]
```diff
@@ -47,6 +47,78 @@ or about 6 GiB. With on-disk compression, the actual disk usage is likely to be
...
+
+- Using the [`SHOW RANGES`]({% link {{ page.version.version }}/show-ranges.md %}) SQL statement, which lists the above values under the names `live_bytes`, `key_bytes`, and `val_bytes`.
+- In the DB Console, under [**Advanced Debug Page > Even more Advanced Debugging**]({% link {{ page.version.version }}/ui-debug-pages.md %}#even-more-advanced-debugging), click the **Range Status** link, which takes you to a page where the values are displayed in a tabular format like the following: `MVCC Live Bytes/Count | 2.5 KiB / 62 count`.
+
+When data has been deleted for at least the duration specified by [`gc.ttlseconds`]({% link {{ page.version.version }}/configure-replication-zones.md %}#gc-ttlseconds), CockroachDB will consider it eligible for 'garbage collection'. Asynchronously, CockroachDB will perform garbage collection of ranges that contain significant quantities of garbage and delete the garbage. Note that if there are backups or other processes that haven't completed yet but require the data, these processes may prevent the garbage collection of that data by setting a protected timestamp until these processes have completed.
```
Comment (taroface) [inline src/current/v23.2/operational-faqs.md:74]: Maybe? Seems redundant

```suggestion
When data has been deleted for at least the duration specified by [`gc.ttlseconds`]({% link {{ page.version.version }}/configure-replication-zones.md %}#gc-ttlseconds), CockroachDB will consider it eligible for 'garbage collection'. Asynchronously, CockroachDB will perform garbage collection of ranges that contain significant quantities of garbage. Note that if there are backups or other processes that haven't completed yet but require the data, these processes may prevent the garbage collection of that data by setting a protected timestamp until these processes have completed.
```

File: src/current/v23.2/operational-faqs.md [hunk 2]
```diff
@@ -47,6 +47,78 @@ or about 6 GiB. With on-disk compression, the actual disk usage is likely to be
...
+    ~~~
+
+1. Drop the table using [`DROP TABLE`]({% link {{ page.version.version }}/drop-table.md %}):
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+DROP TABLE t;
+~~~
+
+1. Visit the [Advanced Debug page]({% link {{ page.version.version }}/ui-debug-pages.md %}) and click the link **Run a range through an internal queue** to visit the **Manually enqueue range in a replica queue** page. On this page, select **mvccGC** from the **Queue** dropdown and enter each of range ID from the previous step. Check the **SkipShouldQueue** checkbox to speed up the MVCC [garbage collection]({% link {{ page.version.version }}/architecture/storage-layer.md %}#garbage-collection) process.
```
Comment (taroface) [inline src/current/v23.2/operational-faqs.md:118]:
```suggestion
1. Visit the [Advanced Debug page]({% link {{ page.version.version }}/ui-debug-pages.md %}) and click the link **Run a range through an internal queue** to visit the **Manually enqueue range in a replica queue** page. On this page, select **mvccGC** from the **Queue** dropdown and enter each range ID from the previous step. Check the **SkipShouldQueue** checkbox to speed up the MVCC [garbage collection]({% link {{ page.version.version }}/architecture/storage-layer.md %}#garbage-collection) process.
```

**Review by taroface — APPROVED** [review]
Looks great!

File: src/current/v23.2/operational-faqs.md [hunk 3]
```diff
@@ -47,6 +47,78 @@ or about 6 GiB. With on-disk compression, the actual disk usage is likely to be
...
 
+## Why is my disk usage not decreasing after deleting data?
+
+{% comment %}
+The below is a lightly edited version of https://stackoverflow.com/questions/74481018/why-is-my-cockroachdb-disk-usage-not-decreasing
+{% endcomment %}
+
+There are several reasons why disk usage may not decrease right after deleting data:
+
+- [The data could be preserved for MVCC history](#the-data-could-be-preserved-for-mvcc-history)
```
[the remark this answers is not in the document]
Comment (rmloveland) [reply src/current/v23.2/operational-faqs.md:58]: ✅

File: src/current/v23.2/operational-faqs.md [hunk 4]
```diff
@@ -47,6 +47,78 @@ or about 6 GiB. With on-disk compression, the actual disk usage is likely to be
...
+- [The data could be in the process of being compacted](#the-data-could-be-in-the-process-of-being-compacted)
```
[the remark this answers is not in the document]
Comment (rmloveland) [reply src/current/v23.2/operational-faqs.md:59]: ✅

File: src/current/v23.2/operational-faqs.md [hunk 5]
```diff
@@ -47,6 +47,78 @@ or about 6 GiB. With on-disk compression, the actual disk usage is likely to be
...
+
+{% include {{page.version.version}}/storage/free-up-disk-space.md %}
+
+### The data could be preserved for MVCC history
+
+CockroachDB implements [Multi-Version Concurrency Control (MVCC)]({% link {{ page.version.version }}/architecture/storage-layer.md %}#mvcc), which means that it maintains a history of all mutations to a row. This history is used for a wide range of functionality: transaction isolation, historical [`AS OF SYSTEM TIME`]({% link {{ page.version.version }}/as-of-system-time.md %}) queries, [incremental backups]({% link {{ page.version.version }}/take-full-and-incremental-backups.md %}), [changefeeds]({% link {{ page.version.version }}/create-and-configure-changefeeds.md %}), [cluster replication]({% link {{ page.version.version }}/architecture/replication-layer.md %}), and so on. The requirement to preserve history means that CockroachDB "soft deletes" data: The data is marked as deleted by a tombstone record so that CockroachDB will no longer surface the deleted rows to queries, but the old data is still present on disk.
```
[the remark this answers is not in the document]
Comment (rmloveland) [reply src/current/v23.2/operational-faqs.md:65]: 🔗 YESSSS I LOVE IT 🔗

File: src/current/v23.2/operational-faqs.md [hunk 6]
```diff
@@ -47,6 +47,78 @@ or about 6 GiB. With on-disk compression, the actual disk usage is likely to be
...
+
+The length of history preserved by MVCC is determined by two things: the [`gc.ttlseconds`]({% link {{ page.version.version }}/configure-replication-zones.md %}#gc-ttlseconds) of the zone that contains the data and whether any [protected timestamps]({% link {{ page.version.version }}/architecture/storage-layer.md %}#protected-timestamps) exist. You can check the range's statistics to observe the `key_bytes`, `value_bytes` and `live_bytes`. The `live_bytes` metric reflects data that's not garbage. The value of (`key_bytes` + `value_bytes`) - `live_bytes` will tell you how much MVCC garbage is resident within a range.
```
[the remark this answers is not in the document]
Comment (rmloveland) [reply src/current/v23.2/operational-faqs.md:67]: ✅

File: src/current/v23.2/operational-faqs.md (hunk 1 above)
Comment (rmloveland) [reply src/current/v23.2/operational-faqs.md:74]: ✅

File: src/current/v23.2/operational-faqs.md (hunk 2 above)
Comment (rmloveland) [reply src/current/v23.2/operational-faqs.md:118]: ✅

**Pull request merged**